frontend/src/tests/components/common/LogoutScreen.spec.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { shallow } from "enzyme";
+import { Logout } from "@/components/common/Logout";
+
+const props = {};
+
+const setupProps = () => {
+  props.logoutUser = jest.fn();
+  props.keycloak = {};
+};
+
+beforeEach(() => {
+  setupProps();
+});
+
+describe("LogoutScreen", () => {
```

Comment:
The test should have the same file name as the component being tested.

Reply:
Oh, I missed updating that! It was originally called LogoutScreen. Will update 👍